src/index.ts

```diff
@@ -9,7 +9,7 @@ import { MongoClient } from './mongo_client';
 import { Db } from './db';
 import { Collection } from './collection';
 import { Logger } from './logger';
-import { GridFSBucket } from './gridfs-stream';
+import { GridFSBucket } from './gridfs';
```

Comment:
I can revert this, I figured now that its our only "gridfs" no need to name it after its implementation detail